docs/courses/challenge.md

```diff
@@ -0,0 +1,87 @@
+# LibreLingo Documentation on Challenge
+
+A **Challenge** is a single task or question the user responds to as they study using LibreLingo. Challenges are not configured directly, but [automatically generated from Skill configuration](skill.md#how-skills-generate-challenges).
+
+To get a better understanding, you can read the [course basics](README.md#basics).
+
+---
+
+**Confused?**
+[Ask people](https://github.com/kantord/LibreLingo/discussions) on GitHub Discussions.
+
+---
+
+**Table of Contents:**
+- [Cards challenge](#cards-challenge)
+- [Short input challenge](#short-input-challenge)
+- [Options challenge](#options-challenge)
+- [Chips challenge](#chips-challenge)
+- [Listening challenge](#listening-challenge)
+
+## Cards challenge
+
+<img src="/docs/screenshots/cards-challenge.png" width="400px"  alt="Text asks the user: which of these is \"river\"?. Below this heading are three cards. Each card contains a Spanish word and corresponding image. In the center the correct choice (río) is selected."></img>
```

Comment:
Good job for writing a good alt text and including images! 💯

I wonder though, is it possible to use alt texts with "proper" markdown syntax? 🤔 https://github.github.com/gfm/#images

Reply:
I'll play around with it -- I don't think `width="400px"` is supported by GFM but that might not even be necessary.